config.toml

```diff
@@ -47,7 +47,7 @@ anchor = "smart"
 [services]
 [services.googleAnalytics]
 # Comment out the next line to disable GA tracking. Also disables the feature described in [params.ui.feedback].
-id = "UA-70174036-1"
+id = "G-H0XE7ESBFR"
```

Comment:
Curious where the old and new magic numbers come from.

Reply:
Via the Google Analytics UI 😅  In the previous GA project, I see the old ID associated with the project, in the new GA project that is setup for GA4, there's multiple types of IDs, the ID I used here was the Measure ID

[Old ID Location](https://analytics.google.com/analytics/web/#/a70174036w107369022p111825983/admin/property/settings)

[New ID Location](https://analytics.google.com/analytics/web/#/p379952698/reports/intelligenthome)

Reply:
Might be good to add this as a comment in the config.toml for the future.

Reply:
Yes, it should be the MEASUREMENT ID